Assets/Packages/com.studio23.ss2.inventorysystem/Runtime/Core/InventoryBase.cs

```diff
@@ -106,6 +106,13 @@ public virtual async UniTask LoadInventory(bool enableEncryption = true, string
                 );
 
             _items.Clear();
+
+            if(loadedItemDatas == null )
+            {
+                Debug.LogWarning($"No inventory file found for {InventoryName}, Save Inventory First");
+                return;
+            }
+
             foreach (var loadedItemData in loadedItemDatas)
             {
                 T item = Resources.Load<T>($"Inventory System/{InventoryName}/{loadedItemData.SOName}");
```

Comment:
If we are going to add null warnings, there should be one when this fails.
#8

Reply:
That's another issue. I'll make another pr for that one. You created seperate issues for the same method.